test/unit/ts-target-es2018/output-coverage.js

```diff
@@ -0,0 +1,16 @@
+/******/ (() => { // webpackBootstrap
+/******/ 	"use strict";
+/******/ 	/* webpack/runtime/compat */
+/******/ 	
+/******/ 	if (typeof __nccwpck_require__ !== 'undefined') __nccwpck_require__.ab = __dirname + "/";
+/******/ 	
+/************************************************************************/
+var __webpack_exports__ = {};
+
+if (process === null || process === void 0 ? void 0 : process.env.FOO) {
+    console.log('foo');
+}
```

Comment:
@TooTallNate Here we can see the optional chaining operator is properly down-leveled 😌